WasatchNET/EEPROM.cs

```diff
@@ -1936,20 +1989,23 @@ protected bool writeParse()
                     if (!ParseData.writeByte(matchingMinRampPixels,     pages[7], 4)) return false;
                     if (!ParseData.writeUInt16(matchingMinPeakHeight,   pages[7], 5)) return false;
                     if (!ParseData.writeByte(matchingThreshold,         pages[7], 7)) return false;
-
+                    if (!ParseData.writeByte(librarySize,               pages[7], 8)) return false;
+                    int names_written = 0;
+                    int name_page;
                     if (librarySpectrum == null)
                     {
                         logger.error("EEPROM.writeLibrary: no librarySpectrum");
                         return false;
                     }
 
-                    if (librarySpectrum.Count > 2048)
+                    if (librarySpectrum.Count > PIXELS*MAX_LIB_ENTRIES)
                     {
-                        logger.error("EEPROM.writeLibrary: librarySpectrum only sized for 2048 pixels");
+                        logger.error("EEPROM.writeLibrary: librarySpectrum only sized for 8 spectra of size 1952 pixels");

```

Comment:
if we do use the pre-existing pixels attribute we'll want to add some intelligence most likely. And use our constants